### ISSUE TITLE
Cancel object from print when one object fails but others are fine

### ISSUE THREAD

**Issue by Dpariet** [bambulab/BambuStudio#4819]
**Is your feature request related to a problem? Please describe.**
I am printing an assembly of different parts and have 4 different objects printing on the same plate.  1 object has failed and the other 3 are printing perfectly.  I am now wasting all the filament and time on the failed object.

**Describe the solution you'd like**
On bambu studio or the handy mobile app, when you start a print have a screen that shows the print bed and objects being printed and have the ability to cancel an object out of the print if it fails sometime during the print.  (Sonic Pad has this feature)

**Describe alternatives you've considered**
This is the only efficient solution

**Additional context**

**Comment by DrymarchonShaun** [comment]
From the way you worded it, I can't tell if you are aware or not that this *can* be done from the mobile app - https://wiki.bambulab.com/en/general/skipping-objects#operate-on-the-app

**Comment by Dpariet** [comment]
I did not know that, bambu studio should get this feature also!

**Comment by thebadluck** [comment]
I agree this feature would be very helpful in Bambu Studio.  I have a P1S so the screen is useless for most anything.   I do not always have my phone handy when I am working at my computer.  Even when I do, it takes time to open the app.  I have seen this feature requested several times.

**Comment by jlo1975** [comment]
we can do everything in bambu studio except this thing.....
So, yes not have the option in bambu studio to mark failed object to be skipped is a huge design mistake.

**Comment by darterdave** [comment]
I just tried to save dozens of hours in a failing print. Used BH app for my P1S and skipped the objects that I wanted to cancel. It was confirmed. And printer continued to try and print the objects that I had just canceled. What a waste of time and money. Thousands of dollars of equipment and I can only abort the entire print job. Please make a solution to this issue.

**Comment by prdoyle** [comment]
Dupe of #3281?
